webapp/channels/src/components/drafts/draft_actions/delete_draft_modal.tsx

```diff
@@ -49,6 +49,8 @@ function DeleteDraftModal({
             onExited={onExited}
             compassDesign={true}
             isDeleteModal={true}
+            autoFocusConfirmButton={true}
+            confirmButtonClassName={'a11y--active a11y--focused'}
```

Comment:
i am not sure if this is required because it would make the button with focused style all the time

Reply:
well, what I understood from PR was-> Every time the delete icon on a draft is clicked, its delete modal pop appears and we want its delete button to be focused everytime. Closing my PR as I didn't get requirements.

Reply:
i mean the buttons should still be tabble with correct focus styling, if we were to force the confirm button with that focused style the tabbing wont correctly appear when we tab. You can just pass`autoFocusConfirmButton` to true and see it that solves the problem.

Reply:
did you signup? if yes whats the error that shows up?

Reply:
while signup "An account with that username already exists". For forgot password I haven't received any email yet to reset.

Reply:
> i mean the buttons should still be tabble with correct focus styling, if we were to force the confirm button with that focused style the tabbing wont correctly appear when we tab. You can just pass`autoFocusConfirmButton` to true and see it that solves the problem.

Ohh, you meant to remove `confirmButtonClassName={'a11y--active a11y--focused'}` so that no visual changes are seen.